includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -236,4 +237,43 @@ private function is_gif_url( $url ) {
 		$path = AMP_WP_Utils::parse_url( $url, PHP_URL_PATH );
 		return substr( $path, -strlen( $ext ) ) === $ext;
 	}
+
+	/**
+	 * Handles an issue with the aligncenter class.
+	 *
+	 * If the <amp-img> has the class aligncenter, this strips the class and wraps it in a <figure> to center the image.
+	 * There was an issue where the aligncenter class overrode the "display: inline-block" rule of AMP's layout="intrinsic" attribute.
+	 * So this strips that class, and instead wraps the image in a <figure> to center it.
+	 *
+	 * @since 0.7
+	 * @see https://github.com/Automattic/amp-wp/issues/1104
+	 *
+	 * @param DOMNode|DOMElement $node The <amp-img> node.
+	 * @return DOMNode|DOMElement $node The <amp-img> node, possibly wrapped in a <figure>.
```

Comment:
There is no need for `DOMNode` in the phpdoc tags here. It will always be a `DOMElement` (and `DOMElement` is a subclass of `DOMNode`).

Reply:
Thanks, [this commit](https://github.com/Automattic/amp-wp/pull/1109/commits/25c2efcb7e0cfbfb77d3f1f6815a34a560760afe) removes `DOMNode` from the tags.